apps/web/vibes/soul/sections/product-detail/product-detail-form.tsx

```diff
@@ -109,6 +111,7 @@ export function ProductDetailForm<F extends Field>({
             <NumberInput
               aria-label={quantityLabel}
               decrementLabel={decrementLabel}
+              disabled={disabled}
```

Comment:
Is it necessary to disable the quantity input as well? That seems reasonable but if we rename to `ctaDisabled` it might be confusing. If we keep the name `disabled`, then do we want to disable all fields in the form? Would that make things feel broken as opposed so just disabling the submit button?

Reply:
I will keep enabled for now and we can revisit.